src/components/LightUpChinatown/LightUpChinatownPage.tsx

```diff
@@ -133,7 +153,48 @@ const LightUpChinatownPage = () => {
     </React.Fragment>
   );
 };
+// left: 42%;
+// top: 70%;
+const Button = styled.div`
+  line-height: 40px;
+  width: 15%;
+  min-width: 150px;
+  max-width: 220px;
+
+  position: absolute;
```

Comment:
I'd use flexbox instead of position absolute to center it

Reply:
added flexbox centering in the latest commit